.php_cs

```diff
@@ -38,46 +38,47 @@ $config->setRules(
     [
         '@PSR2' => true,
         'align_multiline_comment' => ['comment_type' => 'all_multiline'],
+        'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'blank_line_before_statement' => [
             'statements' => ['return', 'continue', 'break', 'declare', 'throw', 'try'],
         ],
-        'cast_spaces' => ['space' => 'none'],
-        'combine_consecutive_issets' => true,
-        'combine_consecutive_unsets' => true,
+        'binary_operator_spaces' => true,
+        'cast_spaces' => ['space' => 'single'],
+        'class_attributes_separation' => ['elements' => ['method']],
         'compact_nullable_typehint' => true,
         'concat_space' => ['spacing' => 'one'],
-        'dir_constant' => true,
-        'function_to_constant' => true,
+        'declare_equal_normalize' => true,
+        'explicit_indirect_variable' => true,
+        'explicit_string_variable' => true,
+        'fully_qualified_strict_types' => true,
         'function_typehint_space' => true,
         'heredoc_to_nowdoc' => true,
         'include' => true,
         'linebreak_after_opening_tag' => true,
         'list_syntax' => ['syntax' => 'short'],
         'lowercase_cast' => true,
         'magic_constant_casing' => true,
-        'mb_str_functions' => true,
```

Comment:
👎

Reply:
Forcing `mb_` has nothing to do with formatting, is risky, and may introduce unintended behaviour.

Reply:
Only risky where someone has overridden the functions - which is something that would go against the `sane` in `sane-defaults` :)

Coding standards aren't just about formatting :)